navigation-material/src/androidTest/java/com/google/accompanist/navigation.material/BottomSheetNavigatorTest.kt

```diff
@@ -368,6 +388,379 @@ internal class BottomSheetNavigatorTest {
         }
     }
 
+    @Test
+    fun testSheetContentSizeChangeDuringAnimation_opensSheet_shortSheetToShortSheet() {
+        val animationDuration = 2000
+        val animationSpec = tween<Float>(animationDuration)
+        lateinit var navigator: BottomSheetNavigator
+        lateinit var navController: NavHostController
+        var height: Dp by mutableStateOf(20.dp)
+
+        composeTestRule.setContent {
+            navigator = rememberBottomSheetNavigator(animationSpec)
+            navController = rememberNavController(navigator)
+            navController.currentBackStackEntryAsState().value
+            ModalBottomSheetLayout(navigator) {
+                NavHost(navController, "first") {
+                    composable("first") {
+                        Box(Modifier.fillMaxSize())
+                    }
+                    bottomSheet("sheet") {
+                        Box(Modifier.height(height))
+                    }
+                }
+            }
+        }
+
+        composeTestRule.mainClock.autoAdvance = true
+
+        composeTestRule.waitForIdle()
+        composeTestRule.runOnUiThread {
+            navController.navigate("sheet")
+        }
+        composeTestRule.mainClock.advanceTimeBy(100)
+        height = (composeTestRule.onRoot().getUnclippedBoundsInRoot().height) / 3
```

Comment:
Can we verify that the sheet is in `transitionsInProgress` here while the sheet is being opened?

Similarly, it would be good to verify that it has been removed from that set after the animation finishes, here and in the other tests as well.

Reply:
Done